src/Medidata.ZipkinTracer.Core/ZipkinClient.cs

```diff
@@ -20,11 +17,14 @@ public class ZipkinClient : ITracerClient
         private string requestName;
         private ITraceProvider traceProvider;
         private ILog logger;
+        private List<string> internalDomainList;
 
         public ZipkinClient(ITraceProvider tracerProvider, string requestName, ILog logger) : this(tracerProvider, requestName, logger, new ZipkinConfig(), new SpanCollectorBuilder()) { }
 
         public ZipkinClient(ITraceProvider traceProvider, string requestName, ILog logger, IZipkinConfig zipkinConfig, ISpanCollectorBuilder spanCollectorBuilder)
         {
+            internalDomainList = zipkinConfig.GetInternalDomainList();
```

Comment:
@jcarres-mdsol Do we really need the removing internal domain feature? I think full domain and short name which removes "imedidata.{net|com}" are almost same... And maybe it's hard for users to understand "internalDomainList" in config. I prefer simple implementation that uses full domain.